src/systemtest/java/randoop/main/RandoopSystemTest.java

```diff
@@ -113,6 +116,9 @@ public void runCollectionsTest() {
             options,
             timeout);
 
+    long endTime = System.currentTimeMillis();
+    System.out.println("runCollectionTest took: " + (endTime - startTime) + " milliseconds");
```

Comment:
I don't think this should be printed unconditionally.  It's useful for debugging, but I suggest you comment it out.